templates/Datashades-OpsWorks-CKAN-Instances.cfn.yml.j2

```diff
@@ -501,4 +507,94 @@ Resources:
 {% endfor %}
 {% endif %}
 
+{% endfor %}
+
+  # Autoscaling instances
+
+{% for layer in ['Batch', 'Web', 'Solr'] %}
+  {{ layer }}LaunchTemplate:
+    Type: AWS::EC2::LaunchTemplate
+    Properties:
+      LaunchTemplateName: !Sub "${Environment}-${ApplicationName}-{{ layer|lower }}-LaunchTemplate"
+      LaunchTemplateData:
+        BlockDeviceMappings:
+          - DeviceName: "/dev/xvda"
+            Ebs:
+              DeleteOnTermination: true
+              VolumeSize: 100
+              VolumeType: "gp2"
+          - DeviceName: "/dev/sdi"
+            Ebs:
+              DeleteOnTermination: true
+              VolumeSize: 32
+              VolumeType: "gp2"
+        IamInstanceProfile:
+          Arn:
+            Fn::ImportValue: !Sub "${Environment}${ApplicationName}WebInstanceProfile"
+        ImageId: ami-03b836d87d294e89e
```

Comment:
What image is this?

Reply:
It should probably be a parameter, I suppose.

It's a public AMI of Amazon Linux 2 (ie what OpsWorks has been using).